spec/ruby/core/queue/pop_spec.rb

```diff
@@ -1,6 +1,17 @@
 require_relative '../../spec_helper'
 require_relative '../../shared/queue/deque'
+require_relative '../../shared/types/rb_num2dbl_fails'
 
 describe "Queue#pop" do
   it_behaves_like :queue_deq, :pop, -> { Queue.new }
+  it_behaves_like :queue_deq, :shift, -> { Queue.new }
+  it_behaves_like :queue_deq, :deq, -> { Queue.new }
+end
```

Comment:
There are similar spec files for aliases:
- `spec/ruby/core/queue/deq_spec.rb`
- `spec/ruby/core/queue/shift_spec.rb`

So calls of the shared example below:
```ruby
    it_behaves_like :rb_num2dbl_fails, nil, -> v { q = Queue.new; q.push(1); q.shift(timeout: v) }
    it_behaves_like :rb_num2dbl_fails, nil, -> v { q = Queue.new; q.push(1); q.deq(timeout: v) }
```
are supposed to be located in these files.

Reply:
Aah that's why I missed them (original aliases). Thanks I'll move them.